core/templates/dev/head/domain/exploration/EditableExplorationBackendApiServiceSpec.js

```diff
@@ -55,7 +55,7 @@ describe('Editable exploration backend API service', function() {
             default_outcome: {
               param_changes: [],
               dest: 'Introduction',
-              feedback: []
+              feedback: null
```

Comment:
Is this actually possible in practice? I thought that, if default_outcome is not null, it would contain a proper feedback object...

Reply:
You're right, changed it to be a feedback object